package.json

```diff
@@ -42,7 +42,8 @@
       ],
       "allowedVersions": {
         "@solana/web3.js": "1.52.0",
-        "ethers": "5.7.2"
+        "ethers": "5.7.2",
+        "jest": "29.4.3"
```

Comment:
why is that?

Reply:
To silence a warning of mismatch of peer dependency from a 3rd party lib. Which is nosense cause we don't even run the tests of the lib, but 🤷